subprojects/packagefiles/freeglut/meson.build

```diff
@@ -67,12 +67,29 @@ freeglut_sources = files(
   'src/fg_window.c',
 )
 
+dep_glu = dependency(
+  'glu',
+  required: false,
+)
+# GLU is part of OpenGL.Framework
+if not dep_glu.found() and host_machine.system() != 'darwin'
```

Comment:
We don't require the dependency on macOS, then?

Reply:
> And on macOS, GLU will be available through `dependency('gl')`

That answers it, thanks.  Might be good to mention that in a comment, since it's not clear by inspection.